helm/prometheus-meta-operator/templates/prometheus-rules/etcd.management-cluster.rules.yml

```diff
@@ -5,22 +5,11 @@ metadata:
   labels:
     {{- include "labels.common" . | nindent 4 }}
     cluster_type: "management_cluster"
-  name: etcd.rules
+  name: etcd.management-cluster.rules
 spec:
   groups:
   - name: etcd
     rules:
-    - alert: BackupJobFailedOrStuck
```

Comment:
Should this be dropped?

Reply:
It is,  I just need to renew the diff

Reply:
Is this a cleanup thing? Feels like dropping rules should be done as a separate PR before moving anything otherwise we loose track (including nothing in CHANGELOG) about what was dropped explicitly, vs for example mysteriously going missing during migration.

Reply:
Technically, I did not drop it but this PR has some etcd rule for workload cluster only so I split this etcd file into 2:
etcd.all.rules.yaml and etcd.workload_cluster.rules.yaml. Also,  I renamed the rule to ETCDBackupJobFailedOrStuck but this alert was added in the context of this PR so announcing it in the changelog did not seem useful to me

Sorry for the confusion

Reply:
Do you want to sync on this so I can improve how I migrate rules? Otherwise, I could use another review :D

Reply:
Ah, I see so it was renamed and moved, I didn't notice that, all good then, thanks for explaining. As for CHANGELOG, yeah should be OK then, maybe just adding a point about the added/renamed rule, I added a suggestion above, but it's optional sure.

Reply:
I committed your suggestion :)